### ISSUE TITLE
Arethusa does not seem to be able to identify verb forms which finish in an apostrophe (Greek)

### ISSUE THREAD

**Issue by KateCook1** [perseids-project/TeachtheTeachers#1]
IIE - the morphology tagging does not work. Has anyone else seen this and/or know how to solve it?

**Comment by Dimitar-Iliev** [comment]
You can add them as new forms in the morphological tab.

**Comment by KateCook1** [comment]
Yes, but this is time consuming and not ideal for student work where there is a higher chance that they make mistakes. 

I'm basically just wondering if (given that Perseus CAN identify these forms via the word study tool) there's a quicker way to sort this out as a general issue... @TDBuck

**Comment by Dimitar-Iliev** [comment]
That's true. But, on the other hand, my students often get confused when they see apostrophed forms such as e.g. (ἐ)γένετ' and cannot identify the ending with the missing vowel. I think it could help if they get used to seeing such forms in the list on the right. They might start recognizing them more easily.

**Comment by TDBuck** [comment]
@KateCook1 I think that making students fill out morphological forms is \* always \* a good thing. They might make a mistake, but that is alright. The new form the create is local to their document, and doesn't feed back into Morpheus (the morphological processor). It means you might need to watch their morphological tags when grading the trees, but you should be doing that anyway, since they can always accidentally select the wrong autoparsed form.

**Comment by balmas** [comment]
also, note that if you have the "Local Storage" setting checked (click the little settings wheel on the morph tab and check "Local Storage") then when you input new forms, they will be remembered in your browser's cache, and come up the next time you encounter the word. In addition, if you have the "Store Preferences" option checked, the frequency with which you choose one form over another will be recorded as you annotated (again in your browser's cache) and after enough annotating, you will begin to see your most frequent choices for forms listed first.

**Comment by nevenjovanovic** [comment]
@balmas , could the browser cache / local storage be exported and shared across computers? That way the students could import the teacher's cache into their own browser instances, and that would be a local way to solve apostrophes problems. Perhaps the process could eventually be automated, our the parses could be imported into the official Arethusa parser... is it doable? Would it be practical?

**Comment by balmas** [comment]
Yes, that is already possible and it was seen as a stepping stone to having a centralized place to capture this information.

Click the Help (?) menu, then the Tools link, then Import/Export your morphological data.

**Comment by nevenjovanovic** [comment]
@balmas I have to quote: "awesome!" Will test it asap!
